bpf/include/bpf_log.h

```diff
@@ -54,4 +83,70 @@ reason is direct access would not be print ipaddr when pass `&ctx->remote_ipv4`
     __u32 name = 0;                                                                                                    \
     name = ctx_ip;
 
+/* Add KERNEL_VERSION_HIGHER_5_13_0 to resolve problem, when kernel version lower than 5.13, or linux distribution lower
+than 22.09, compile would report an error which bpf_snprintf dont exist */
+#if KERNEL_VERSION_HIGHER_5_13_0
+#define Kmesh_BPF_SNPRINTF(out, out_size, fmt, args...)                                                                \
```

Comment:
The current implementation may be complex.
IMO, we could implement `ip2str` function and  print ip address based on strings. for example:
```c
char * ip2str(u32 *ip, bool is_v4) {
// fmt ip to string
}

BPF_LOG("ip:%s, port:%d\n", ip2str(&ip, true), port);
```

Reply:
Using ip2str maybe cause whatever log level will execute ip convert logic. Do we
need to control it by log level?